openapi/platform/platform.yaml

```
@@ -2863,6 +2863,185 @@ paths:
       tags:
         - Options
         - User Custom Fields
+  '/productCustomFields/{id}':
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+        example: ABC-MP-1234567890ABCDEFGHIJKLMNOPQRSTUV
+        description: The ID of the Custom Fields - actually the same id of Product
```

This phrasing took me a few times to understand. How about this?
```suggestion
        description: The custom fields, for the product with this ID.
```

---

I changed to ` description: The ID of the Custom Fields, which is a Product ID`, to match watch was suggested in the `api-gateway`